Makefile

```diff
@@ -160,10 +166,16 @@ HIPCCFLAGS ?= $(filter-out $(OMP_SIMD_FLAG),$(OPT)) -fPIC -munsafe-fp-atomics
 ifneq ($(HIP_ARCH),)
   HIPCCFLAGS += --amdgpu-target=$(HIP_ARCH)
 endif
-
 SYCL_FLAG := $(SYCL_FLAG.$(CC_VENDOR))
 SYCLFLAGS ?= $(SYCL_FLAG) -fPIC -std=c++17 $(filter-out -std=c++11,$(CXXFLAGS)) $(filter-out $(OMP_SIMD_FLAG),$(OPT))
-FFLAGS ?= $(OPT) $(FFLAGS.$(FC_VENDOR))
+
+OPENMP ?=
+ifneq ($(OPENMP),)
+  OMP_FLAG := $(OMP_FLAG.$(CC_VENDOR))
+  OMP_FLAG := $(if $(call cc_check_flag,$(OMP_FLAG)),$(OMP_FLAG))
+  CFLAGS += $(OMP_FLAG) -DCEED_USE_OPENMP
```

Comment:
This is a preprocessor flag and thus belongs in `CPPFLAGS`, but I'm not sure it's needed. Why not use `_OPENMP` as specified by the standard? Is the concern that the compiler may activate it by default, but we may not want to use atomics unless the user really wants to?

Reply:
Whoops, of course it's better to just use `_OPENMP`. I wasn't aware this was actually in the standard and consistent across implementations, thanks!